vtysh/bgp_vty.c

```diff
@@ -5117,16 +5155,26 @@ cli_neighbor_timers_execute(char *vrf_name, int argc, const char *argv[])
     if (!bgp_router_context) {
         ERRONEOUS_DB_TXN(txn, "bgp router context not available");
     }
-        ovs_bgp_neighbor =
+
+    ovs_bgp_neighbor =
     get_bgp_neighbor_with_bgp_router_and_ipaddr(bgp_router_context, ip_addr);
 
     if (ovs_bgp_neighbor) {
-        key_timers[0] = OVSDB_BGP_TIMER_KEEPALIVE;
-        key_timers[1] = OVSDB_BGP_TIMER_HOLDTIME;
-        /* To write to ovsdb nbr table. */
-        ovsrec_bgp_neighbor_set_timers(ovs_bgp_neighbor, key_timers,
-                                       (int64_t *)&tim_val, 2);
+        if (ovs_bgp_neighbor->bgp_peer_group) {
+            VLOG_ERR(OVSDB_TXN_COMMIT_ERROR);
+            VLOG_ERR("Error in transaction for setting timers of neighbor."
+                     "Neighbor %s already has been assigned to the peer group.\n",
+                                                                        argv[0]);
```

Comment:
propose exclude macros VLOG_ERR.. Logging with level of DBG is already in ERRONEOUS_DB_TX macro.